src/Core/Services/Implementations/AzureQueueService.cs

```diff
@@ -21,7 +21,7 @@ protected AzureQueueService(QueueClient queueClient, JsonSerializerOptions jsonO
 
         public async Task CreateAsync(T message)
         {
-            var json = JsonSerializer.Serialize(message, _jsonOptions);
+            var json = JsonSerializer.Serialize(new [] { message }, _jsonOptions);
```

Comment:
I'd rather get rid of the `CreateAsync` method entirely

Reply:
I just made a change, is that what you mean or do you also want to remove it from `IEventWriteService` and make each call of `CreateAsync` call `CreateManyAsync` too?

Reply:
Yes, we should get rid of the single method entirely. I'm fine having single log events in the actual event services, but this class should only enqueue arrays and a helper to convert your single message into an array is of limited value.

We also definitely want to remove the complexity from `CreateManyAsync` that defaults to using the `CreateAsync` method for arrays of length == 1

Reply:
Okay, then I think this has what you want.